src/components/SchedulePage/Event.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+
+import { Typography } from '@material-ui/core';
+
+const styles = theme => ({
+	eventName: {
+		fontFamily: theme.typography.fontFamily,
+		color: theme.palette.secondary.main,
+		fontWeight: 'bold',
+		textTransform: 'uppercase'
+	},
+	eventLocation: {
+		fontFamily: theme.typography.fontFamily,
+		fontWeight: theme.typography.fontWeightMedium
+	}
+});
+
+function Event(props) {
+	return (
+		<Container maxWidth="md">
+			<Grid container spacing={1}>
+				<Grid item xs={12} sm={12} md={4}>
```

Comment:
I suspect that we could either leave off `xs={12}`, or just say `xs`, to make the `sm` value apply to `xs` automatically. Could you experiment a bit here and tell us what you find?

Reply:
It looks like leaving off `xs={12}` works. It will use the value for given for `sm` when it passes the threshold for `xs`. Additionally, as a less relevant note, it seems that if you specify something like `xs={12} md={4}`, it will use the value for `xs` when going lower than the threshold for `md`. So I think even using that would work. I'm unsure what the convention is.

Reply:
It's up to you, though here are some of my thoughts.

To me, specifying `sm` and `md` would be more illuminating, as there is no question that `xs` would follow `sm`. On the other hand, specifying `xs` and `md` would leave me wondering what `sm` would be like: `xs` or `md`?